packages/core/src/components/ColorPicker/Picker/Picker.styles.tsx

```diff
@@ -1,11 +1,8 @@
-import { CSSInterpolation } from "@emotion/serialize";
+import { createClasses } from "@core/utils";
 import { theme } from "@hitachivantara/uikit-styles";
 import { CSSProperties } from "react";
-import { HvColorPickerPickerClasses } from "./pickerClasses";
 
-export const styles: Partial<
-  Record<keyof HvColorPickerPickerClasses, CSSInterpolation>
-> = {
+export const { staticClasses, useClasses } = createClasses("HvColorPicker", {
```

Comment:
Nice catch, thanks 👍

I was trying figure out how/where to name these long names, so that the whole styles didn't get indented, and forgot this one. Trying to avoid git diff in every style line